README.md

```diff
@@ -96,5 +96,6 @@ $ echo "your-openweathermap-appid" > .access_tokens/openweathermap
 * Xcode 9
 * iOS 10+
 * Swift 4
+* code
```

Comment:
What does this `code` adds to the "Requirements" section ?

Reply:
thanks @doriandevtech , close it now